xknx/remote_value/remote_value.py

```diff
@@ -182,21 +180,13 @@ async def process(self, telegram: Telegram, always_callback: bool = False) -> bo
                 device_name=self.device_name,
                 feature_name=self.feature_name,
             )
-        _new_payload = self.payload_valid(telegram.payload.value)
-        if _new_payload is None:
-            raise CouldNotParseTelegram(
-                "payload invalid",
-                payload=str(telegram.payload),
-                destination_address=str(telegram.destination_address),
-                source_address=str(telegram.source_address),
-                device_name=self.device_name,
-                feature_name=self.feature_name,
-            )
+
         try:
+            _new_payload = self.payload_valid(telegram.payload.value)
             decoded_payload = self.from_knx(_new_payload)
-        except ConversionError as err:
+        except (ConversionError, CouldNotParseTelegram) as err:
             logger.warning(
-                "Can not process %s for %s - %s: %s",
+                "Can not process %s \n for %s - %s: %s",
```

Comment:
Do we need the newline here? I'd generally try to avoid newlines in log outputs.

Reply:
Oh... I was thinking about adding even more (not here but elsewhere. With proper indentation do our logs become better readable. 

Here the logged Telegram repr is probably wrapping a line.

Reply:
I've reverted this in c41c2a8